cypress/e2e/createEditTenure.feature

```diff
@@ -1,8 +1,11 @@
+@tenure
 @authentication
 @common
-@processes
 @root
-@tenure
+@search
```

Comment:
I see 2 distinct features being smushed together within this spec - it's 1 (create tenure) and 2+3 (sync tenure to search):
1. Actual tenure creation. (tenure MFE)
2. Tenure Sync with the search (which is a BE feature, but can be tested together with 3) (tenure API + tenure Listener)
3. The new tenure Records are searchable (Housing Search MFE)

Spec file is named `createEditTenure.feature` - there's nothing about the sync in it. I'd argue sync is a separate feature all together _(pretty much `record data` not part of `search data` just like GET endpoint work wouldn't be part of POST endpoint work within API)_